dist/Models/board.js

```diff
@@ -0,0 +1,64 @@
+"use strict";
```

Comment:
You should gitignore the `dist` file.

Reply:
The build folder is not required to push in codebase because it's compiled version of your typescript or es6 code.